tempodb/backend/cache/memcached/memcached.go

```diff
@@ -0,0 +1,63 @@
+package memcached
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+
+	cortex_cache "github.com/cortexproject/cortex/pkg/chunk/cache"
+	"github.com/grafana/tempo/tempodb/backend/cache"
+)
+
+type Config struct {
+	ClientConfig cortex_cache.MemcachedClientConfig `yaml:",inline"`
+
+	TTL time.Duration `yaml:"ttl"`
+}
+
+type Client struct {
+	client *cortex_cache.Memcached
+}
+
+func NewCache(cfg *Config, logger log.Logger) cache.Client {
```

Comment:
Let's rename this to `NewClient` since its returning a client. Similarly for Redis.

Reply:
Changed!